cli/ops/fs.rs

```diff
@@ -577,12 +578,9 @@ fn op_realpath(
     debug!("op_realpath {}", path.display());
     // corresponds to the realpath on Unix and
     // CreateFile and GetFinalPathNameByHandle on Windows
-    let realpath = std::fs::canonicalize(&path)?;
-    let mut realpath_str =
-      into_string(realpath.into_os_string())?.replace("\\", "/");
```

Comment:
Sorry for the possibly dumb question. Isn't the cause of the wrong path separator simply the replacement in this line? What is it for?

Reply:
@nayeemrmn good catch, @piscisaureus PTAL

Reply:
I don't know what you guys think is wrong and what you expect.
I expect backslashes. So it was *wrong* and after this patch it is *good*.

Reply:
@piscisaureus Can we fix this by removing the `\\ -> /` replacement (and maybe adapting the trim statement below), instead of bringing in a new dependency?